backend/spec/support/shared_examples/fiveable.rb

```diff
@@ -0,0 +1,8 @@
+require "spec_helper"
+
+shared_examples_for "fiveable" do
+  include Mongoid::Matchers
+
+  it { is_expected.to respond_to(:value) }
+  it { is_expected.to validate_inclusion_of(:value).to_allow(0..4) }
+end
```

Comment:
Conditional validate in `app/models/checkin/fiveable.rb` concern: `validates :value, inclusion: {in: (0..4)}, if: -> { value.present? }` was the problem. I created this shared example while trying to fix it.  

I like the `it_behaves_like` pattern for concerns, but can revert this shared example change if needed.

Reply:
looks good!